### ISSUE TITLE
Default no friction for first screen or two?

### ISSUE THREAD

**Issue by ariel-phet** [phetsims/masses-and-springs#145]
@arouinfar damping seems like a more advanced topic, so I wonder if for the first screen or two we should default to no friction? Perhaps we just have a check box, much like in projectile motion that give you friction or no friction (with the default as friction off, and then choose some moderate damping for the on mode). I think for the learning goals that I would use such a sim in intro physics on the first two screens (discussing amplitude and such), no friction would be much easier.

**Comment by arouinfar** [comment]
@ariel-phet that appears to be a bug. There isn't a friction slider on the first two screens, so friction _should_ be zero. I think learning goals related to damping are better served on Energy and Lab, so I would not advocate for a friction checkbox on Intro. I could consider allowing damping on Vectors, though it may muddy the waters a bit.

Actually, I'm wondering if we'd be better off renaming "Friction" to "Damping". In the model, the friction slider is adjusting the damping coefficient. I have never heard damping referred to as "friction" in the context of an oscillating spring. If the worry is that "friction" is more familiar to younger students, I would like to point out that WOAS (which gets a lot of use at younger grade levels) has a "damping" slider.

In contrast, the friction slider in Pendulum Lab sets the drag coefficient for the air resistance term of the equation of motion, so "friction" is not inappropriate there.

**Comment by ariel-phet** [comment]
@arouinfar I completely concur on calling it "damping" since that is absolutely the correct term

Even though friction should be zero (I agree that is the best default), I do wonder if like projectile motion we want to communicate that in some way. That is why I mentioned maybe having a "damping" checkbox. Sort of "damping on or off" the same way we have air resistance on or off in projectile motion.  And then on later screens it is adjustable. It seems that works well in projectile motion to communicate the default is no air resistance. We could also wait for interviews to see if that causes any confusion.

**Comment by ariel-phet** [comment]
@Denz1994 be aware that the model should be set to no damping on the first two screens

**Comment by Denz1994** [comment]
Okay, this makes sense. I will work on removing friction from the intro and vector model. Also I can push a dev version with a revised slider entitled "Dampening"

**Comment by ariel-phet** [comment]
@Denz1994 the slider title should just be "Damping" not "Dampening"

**Comment by arouinfar** [comment]
Thanks for the clarification @ariel-phet. I hadn't thought about the damping checkbox reinforcing that damping is off by default, and it's an interesting idea. 

In contrast to air resistance, damping has three states: underdamped, overdamped, and critically damped. I wonder if users would expect a particular kind of damping. I suppose we'd set the damping such that all masses are somewhat underdamped. As a side note, the damping slider appears to control c, not the damping coefficient gamma (gamma = c/2m).

I've been thinking it over a bit, and I think that the constraints of a checkbox may be a better introduction to damping. With a checkbox, students will explore damping on vs. off. With a slider, students may first decide to vary damping rather than focus on on vs. off.

I think adding a "Damping" checkbox under the gravity slider on Intro and Vectors could work nicely. It might be a bit of a squeeze on Vectors, but doable with some pixel polishing. Setting the damping slider to ~25% looks pretty good for a range of different masses.  How does that sound @ariel-phet?

**Comment by ariel-phet** [comment]
@arouinfar yes I think underdamped would be the way to go. Probably good to make a mockup

**Comment by arouinfar** [comment]
Discussed in the 12/21/17 design meeting.

On the Vectors screen, we would need to include a damping force, which is constantly changing (bounded by exponential envelope). This representation would be quite complicated, and is not something typically covered, so we decided not to include damping.

Since Vectors will not include damping, it would be a bit odd flow-wise to include damping on the Intro screen. Other sims, such as ESPB and FAMB, do not introduce friction on the first screen, so it would not be unreasonable to leave damping off the Intro screen.

On Energy and Lab, a damping slider is appropriate, though it should be snappy (like the Friction slider in PL) so that experiments are repeatable. By default, the damping should be set to "none".

To summarize:
* Intro & Vectors: No damping
* Energy & Lab: Snappy damping slider ranging "none" to "lots" (similar to Friction in PL), set to "none" by default.

**Comment by arouinfar** [comment]
@Denz1994, I think it makes sense to move the remaining work to a new issue, as referenced above. Closing.
